### PR TITLE
specifies required node engine in package.json

### PR DESCRIPTION
**Pull request by edm00se**
Sets the minimum node engine in `package.json` to ~~`7.8.0`~~ `7.6.0`.

ref: #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "name": "open-pip",
   "version": "3.0.5",
   "main": "index.js",
+  "engines" : {
+    "node" : ">=7.8.0"
```
Comment (albinekb) [inline package.json]: actually 7.6.0 is enough?

Comment (edm00se) [reply package.json]: Ah, I should have expanded the node.green table. That is now updated in the PR, but if you'll be swapping in that async-to-gen, it'll be nice and moot.

Comment (albinekb) [reply package.json]: Yeah i'll be adding async-to-gen soon :)

**Comment by albinekb** [comment]
Sorry for the delay! And thanks for the help @edm00se 🎉

**Pull request merged**